kolibri/plugins/learn/assets/src/vue/toolbar/index.vue

```diff
@@ -1,22 +1,16 @@
 <template>
 
-  <div v-bind:class="['toolbar-show', displayToolBar ? 'toolbar-hide' : '' ]" v-show='!searchOpen' >
+  <div v-bind:class="['toolbar', displayToolbar ? 'toolbar-hide' : '']" v-show='!searchOpen'>
     <breadcrumbs class="breadcrumbs"></breadcrumbs>
-    <div :class="{ 'toggle-menu-on' : more }">
-      <label for="chan-select" :class="[ more ? 'label-on' : 'visuallyhidden' ]" >{{ $tr('switchChannels') }}</label>
+      <label for="chan-select" class="visuallyhidden">{{ $tr('switchChannels') }}</label>
       <select
-        class="chan-select"
-        :class="[ more ? 'chan-select-mobile-location' : 'chan-select-location' ]"
-        id="chan-select"
         name="chan-select"
+        class="chan-select"
```

Comment:
Select will need an `id` to match the `for` in `<label for="chan-select">`.

@christianmemije